src/api/client/Claims.ts

```diff
@@ -304,7 +304,8 @@ export class Claims {
 
       let ticker: string | undefined;
 
-      if (type === ScopeType.Ticker) {
+      // prettier-ignore
```

Comment:
why ignore prettier here?

Reply:
prettier moves the // NOSONAR to a new line w/o adding this